src/store/middleware.ts

```diff
@@ -55,7 +55,7 @@ export const locationStateMiddleware: Middleware<
         );
       }
     } else if (app.tenant !== tenant) {
-      store.dispatch(setTenant(tenant, false));
+      store.dispatch(setTenant(tenant, undefined, false));
```

Comment:
What's this?

Reply:
basically if the url changes and u changed tenant, it should change too

Reply:
like if the user goes back from `3ddemo` to `contextualization` this would make sure the state is reflecting such change (from URL since we read in the state from there)